projects/1.19/assets/frozen-up/frozenup/lang/zh_cn.json

```
@@ -0,0 +1,94 @@
+  "advancements.frozenup.obtain_truffle.title" : "松露——从无到有",
+  "advancements.frozenup.obtain_truffle.description" : "获得一块松露",
+  "advancements.frozenup.cloud_jumper_boost.title": "冲向……天空？",
+  "advancements.frozenup.cloud_jumper_boost.description": "与驯鹿一起在空中飞翔",
+ 
+  "frozenup.commands.locate.revamped_igloo": "使用/locate structure frozenup:revamped_igloo",
+ 
+  "enchantment.frozenup.hasty_hooves": "急蹄",
+  "enchantment.frozenup.cloud_jumper": "云端跃者",
+ 
+  "item.frozenup.pinecone": "松果",
+  "item.frozenup.compacted_snow_brick": "压实雪砖",
+  "item.frozenup.reindeer_spawn_egg": "驯鹿刷怪蛋",
+  "item.frozenup.penguin_spawn_egg": "企鹅刷怪蛋",
+  "item.frozenup.hoof_armor": "蹄甲",
+  "item.frozenup.frozenup": "千里冰封",
+  "item.frozenup.frozen_truffle": "冰冻松露",
+  "item.frozenup.truffle": "松露",
+  "item.frozenup.truffle_muffin": "松露松饼",
+  "item.frozenup.chilloo_spawn_egg": "寒猹刷怪蛋",
+  "item.frozenup.chilloo_feather": "寒猹羽毛",
+  "item.frozenup.empty_mug": "空马克杯",
+  "item.frozenup.mug_of_chocolate_milk": "杯装热巧克力",
+  "item.frozenup.mug_of_truffle_hot_chocolate": "杯装松露热巧克力",
+  "item.frozenup.mug_of_milk": "杯装奶",
+ 
+  "block.frozenup.compacted_snow_bricks": "压实雪砖",
+  "block.frozenup.compacted_snow_brick_wall": "压实雪砖墙",
+  "block.frozenup.compacted_snow_brick_slab": "压实雪砖台阶",
+  "block.frozenup.compacted_snow_brick_stairs": "压实雪砖楼梯",
+  "block.frozenup.compacted_snow_foundation": "压实雪制地基",
+ 
+  "block.frozenup.compacted_snow_brick_vertical_slab": "压实雪砖竖直台阶",
+  "block.frozenup.cut_ice_vertical_slab": "切制冰竖直台阶",
+  "block.frozenup.cut_ice_cube_vertical_slab": "切制冰块竖直台阶",
+ 
+  "block.frozenup.cut_ice": "切制冰",
+  "block.frozenup.cut_ice_stairs": "切制冰楼梯",
+  "block.frozenup.cut_ice_slab": "切制冰台阶",
+  "block.frozenup.cut_ice_cubes": "切制冰块",
+  "block.frozenup.cut_ice_cube_stairs": "切制冰块楼梯",
+  "block.frozenup.cut_ice_cube_slab": "切制冰块台阶",
+  "block.frozenup.cut_ice_cube_wall": "切制冰块墙",
+  "block.frozenup.chilloo_feather_block": "寒猹羽毛块",
+  "block.frozenup.chilloo_feather_block_carpet": "寒猹羽毛地毯",
+  "block.frozenup.truffle_cake": "松露巧克力蛋糕",
+  "block.frozenup.chilloo_feather_lamp": "寒猹羽毛灯",
+  "block.frozenup.empty_mug": "空马克杯",
+  "block.frozenup.mug_of_milk": "杯装奶",
+  "block.frozenup.mug_of_chocolate_milk": "杯装热巧克力",
+  "block.frozenup.mug_of_truffle_hot_chocolate": "杯装松露热巧克力",
+ 
+  "entity.frozenup.reindeer": "驯鹿",
+  "entity.frozenup.penguin": "企鹅",
+  "entity.frozenup.chilloo": "寒猹",
+   
+  "subtitles.entity.frozenup.reindeer.gallop": "驯鹿：肌腱咔哒声",
+  "subtitles.entity.frozenup.reindeer.breathe": "驯鹿：喘息",
+  "subtitles.entity.frozenup.reindeer.idle": "驯鹿：呼噜声",
+  "subtitles.entity.frozenup.reindeer.hurt": "驯鹿：受伤",
+  "subtitles.entity.frozenup.reindeer.death": "驯鹿：死亡",
+  "subtitles.entity.frozenup.reindeer.angry": "驯鹿：愤怒",
+  "subtitles.entity.frozenup.reindeer.eat": "驯鹿：大声咀嚼",
+  "subtitles.entity.frozenup.reindeer.jump": "驯鹿：跳跃",
+  "subtitles.entity.frozenup.reindeer.cloud_jump": "驯鹿：云跃",
+  "subtitles.entity.frozenup.penguin.idle": "企鹅：尖锐地叫",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "subtitles.entity.frozenup.penguin.idle": "企鹅：嘶鸣",
```

总觉得在说马叫，看着办吧

原文：**Penguin squawks**

---

这个位置就是企鹅在一般情况下的叫声